tools/env/fw_env.config

```diff
@@ -9,6 +9,6 @@
 # Block devices for CompuLab imx8m-plus
 # emmc
 /dev/mmcblk2boot0	0x3f0000 0x4000
-#/dev/mmcblk2boot1	0x3f0000 0x4000
+/dev/mmcblk2boot1	0x3f0000 0x4000
```

Comment:
Even though 'mmcblk2boot0' and 'mmcblk2boot1' have the same address, this is mapped by hardware to two different hardware regions?

Reply:
A note to the redundant setting. From Kconfig:

```
In case CONFIG_SYS_MMC_ENV_PART is 1 (i.e. environment in eMMC boot
partition) then setting CONFIG_ENV_OFFSET_REDUND to the same value
as CONFIG_ENV_OFFSET makes use of the second eMMC boot partition for
the redundant environment copy.
```